roles/init_pd_topo/templates/init_pd_topo.sh.j2

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+{% set all_pd = [] -%}
+{% set pd_hosts = groups.pd_servers %}
+{% for host in pd_hosts -%}
+  {% set pd_ip = hostvars[host].ansible_host | default(hostvars[host].inventory_hostname) -%}
+  {% set pd_port = hostvars[host].pd_client_port -%}
+  {% set pd_path = hostvars[host].deploy_dir -%}
+  {% set _ = all_pd.append("%s:%s%s" % (pd_ip, pd_port, pd_path)) -%}
+{% endfor -%}
+{% set all_tidb = [] -%}
+{% set tidb_hosts = groups.tidb_servers %}
+{% for host in tidb_hosts -%}
+  {% set tidb_ip = hostvars[host].ansible_host | default(hostvars[host].inventory_hostname) -%}
+  {% set tidb_port = hostvars[host].tidb_port -%}
+  {% set tidb_path = hostvars[host].deploy_dir -%}
+  {% set _ = all_tidb.append("%s:%s%s" % (tidb_ip, tidb_port, tidb_path)) -%}
+{% endfor -%}
+{% set all_grafana = [] -%}
+{% set grafana_hosts = groups.grafana_servers %}
+{% for host in grafana_hosts -%}
+  {% set grafana_ip = hostvars[host].ansible_host | default(hostvars[host].inventory_hostname) -%}
+  {% set grafana_port = hostvars[host].grafana_port -%}
+  {% set grafana_path = hostvars[host].deploy_dir -%}
+  {% set _ = all_grafana.append("%s:%s%s" % (grafana_ip, grafana_port, grafana_path)) -%}
+{% endfor -%}
+{% set all_alertmanager = [] -%}
+{% set alertmanager_hosts = groups.alertmanager_servers %}
+{% for host in alertmanager_hosts -%}
+  {% set alertmanager_ip = hostvars[host].ansible_host | default(hostvars[host].inventory_hostname) -%}
+  {% set alertmanager_port = hostvars[host].alertmanager_port -%}
+  {% set alertmanager_path = hostvars[host].deploy_dir -%}
+  {% set _ = all_alertmanager.append("%s:%s%s" % (alertmanager_ip, alertmanager_port, alertmanager_path)) -%}
+{% endfor -%}
+{% set flag = "" %}
+{% if all_tidb -%}
+  {% set flag = flag + "--tidb " + ','.join(all_tidb) -%}
+{% endif -%}
+{% if all_grafana -%}
+  {% set flag = flag + " --grafana " + ','.join(all_grafana) -%}
+{% endif -%}
+{% if all_alertmanager -%}
+  {% set flag = flag + " --alertmanager " + ','.join(all_alertmanager) -%}
+{% endif -%}
+
+python2 init-pd-topo.py --pd {{ all_pd | join(',') }} {{ flag }}
```

Comment:
how about tikv deploy dir?

Reply:
Currently, `init-pd-topo.py` script does not support TiKV.

Reply:
@mapleFU Please figure out a way to support TiKV.

Reply:
Done